package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zkapp-cli",
-  "version": "0.21.3",
+  "version": "0.21.4",
```

Comment:
Just a note to be careful about which version you are updating to because I believe you are updating to the same version in another PR. Maybe you can just land your other approved PRs and bump the version in just one.